server/app/views/applicant/IneligibleBlockView.java

```diff
@@ -114,6 +117,11 @@ public Content render(
             .addMainStyles(ApplicantStyles.MAIN_APPLICANT_INFO)
             .addMainContent(h1(title).withClasses("sr-only"), content);
 
-    return layout.renderWithNav(request, applicantName, messages, bundle);
+    return layout.renderWithNav(
+        request,
+        ApplicantDisplayLabel.ofLoggedInUser(
```

Comment:
TODO(me): can we assume the applicant that the TI is applying for is "logged in"?